OpenRA.Game/GameRules/Ruleset.cs

```diff
@@ -204,10 +204,13 @@ public void LoadActorTraitsFromRuleFile(World world, ModData modData, string[] r
 		{
 			if (ruleFiles == null || ruleFiles.Length == 0)
 			{
-				Console.WriteLine("No rule file specified, reloading all rule files.");
+				TextNotificationsManager.Debug("No rule file specified, reloading all rule files.");
 				ruleFiles = modData.Manifest.Rules;
 			}
+			else
+				TextNotificationsManager.Debug($"Reloading rule files: {string.Join(", ", ruleFiles)}");
```

Comment:
Can you do this for line 278 of Ruleset.cs as well for `LoadWeaponsFromFile`, and finally change line `138 in SequenceSet.cs` to be equivalent to this?

```C#
	if (sequencesFiles == null || sequencesFiles.Length == 0)
	{
		TextNotificationsManager.Debug("No sequence file specified, reloading all sequence files.");
		sequencesFiles = modData.Manifest.Sequences;
	}
	else
		TextNotificationsManager.Debug($"Reloading sequence files: {string.Join(", ", sequencesFiles)}");
```